plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -203,4 +211,52 @@ public static String convertToJson(String yamlString) {
         JSONObject jsonObject=new JSONObject(map);
         return jsonObject.toString();
     }
+
+    /**
+     * Returns the JSON V7 schema for the running jenkins instance.
+     * Example Usage:
+     *      * <pre>{@code
+     *      * Schema jsonSchema = returnSchema();}
+     *      * </pre>
+     * @return Schema Returns the schema for the current jenkins instance
+     */
+    public static Schema returnSchema() {
+        JSONObject schemaObject = generateSchema();
+        JSONObject jsonSchema = new JSONObject(
+            new JSONTokener(schemaObject.toString()));
+        return SchemaLoader.load(jsonSchema);
+    }
+
+    /**
+     *Validates a given jsonObject against the schema generated for the current live jenkins instance
+     *      * Example Usage:
+     *      *      * <pre>{@code
+     *      *      * assertTrue(validateSchema(jsonSubject));}
+     *      *      * </pre>
+     * @param  jsonSubject The json Object that needs to be validated
+     * @return boolean returns true if its valid else returns false
+     */
+    public static boolean validateSchema(JSONObject jsonSubject) {
+        try {
+            returnSchema().validate(jsonSubject);
+        } catch (Exception ie) {
+            LOGGER.warning(failureMessage);
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * Converts a YAML file into a json object
+     * Example Usage:
+     *           * <pre>{@code
+     *           * JSONObject  jsonObject = convertYamlFileToJson("filename");}
+     *           * </pre>
+     * @param yamlFileName the name of the yaml file that needs to be converted
+     * @return JSONObject pertaining to that yaml file.
+     */
+     public static JSONObject convertYamlFileToJson(String yamlFileName) throws Exception {
```

Comment:
Using `Util.class` is the classloader may work here but it won't work in other plugins, you'll need to do the same thing toStringFromYamlFile does
```suggestion
     public static JSONObject convertYamlFileToJson(Object clazz, String yamlFileName) throws Exception {
```

Reply:
Yeah agreed.